lib/WorseReflection/Tests/Inference/virtual_member/method.test

```diff
@@ -10,6 +10,10 @@ class ReflectionMethod {}
 
 class MemberCollection
 {
+    /**
+    * @method string set(string $name)
+    */
```

Comment:
this is defined in the wrong place right?

Reply:
It was yes, my bad. Moved it outside of class now.